plugins/HHAnalyzer.cc

```diff
@@ -147,8 +148,11 @@ void HHAnalyzer::analyze(const edm::Event& event, const edm::EventSetup&, const
            
     for (unsigned int ilep1 = 0; ilep1 < leptons.size(); ilep1++)
     {
+        if ((leptons[ilep1].isMu && leptons[ilep1].p4.Pt() < m_leadingMuonPtCut) || (leptons[ilep1].isEl && leptons[ilep1].p4.Pt() < m_leadingElectronPtCut)) continue;
+
         for (unsigned int ilep2 = ilep1+1; ilep2 < leptons.size(); ilep2++)
         {
+            if ( (leptons[ilep2].isMu && leptons[ilep2].p4.Pt() < m_subleadingMuonPtCut) || (leptons[ilep2].isEl && leptons[ilep2].p4.Pt() < m_subleadingElectronPtCut)) continue;
```

Comment:
this is redundant with the cuts applied for the filling of the lepton structure no ?